src/ChirpWeb/Program.cs

```diff
@@ -1,37 +1,55 @@
+using Microsoft.AspNetCore.Identity;
 using Microsoft.EntityFrameworkCore;
+using ChirpRepositories;
+using ChirpInfrastructure;
+using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.FileProviders;
 using Microsoft.AspNetCore.Builder;
-using Microsoft.AspNetCore.Identity;
 using Microsoft.Extensions.Configuration;
-using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Data.Sqlite;
 using ChirpCore;
 using ChirpCore.Domain;
 using ChirpCore.DTOs;
-using ChirpRepositories;
-using ChirpInfrastructure;
 
 
+var builder = WebApplication.CreateBuilder(args);
 
 
-var builder = WebApplication.CreateBuilder(args);
+if (Directory.Exists("ChirpDatabaseConnection") || File.Exists("ChirpDatabaseConnection"))
+{
+    //File.Delete("ChirpDatabaseConnection");
+    //string DBFilePath = Path.GetTempPath();
+    //string DBFilePathWithFile = Path.Combine(DBFilePath + "chirp.db");
+    //Directory.CreateDirectory(DBFilePath);
 
-// Add services to the container.
-/*builder.Services.AddRazorPages(options =>
+    //File.Create("ChirpWeb.db");
+}
+else
+{
+    //File.Create("ChirpWeb.db");
+}
+/*
+if ()
 {
-    //Razor pages are in a different folder and therefore we use this customized path
-    options.RootDirectory = "/Pages";
-}); */
+    File.Delete("ChirpDatabaseConnection");
+}*/
 
-builder.Services.AddRazorPages(); // Due to Onion Structure setup the implict path works again (same for staticfiles path)
 
 // Load database connection via configuration, get string of database path from appsettings.json
-string? connectionString = builder.Configuration.GetConnectionString("ChirpDatabaseConnection");
+string? connectionString = builder.Configuration.GetConnectionString("ChirpDatabaseConnection") ?? throw new InvalidOperationException("Connection string 'ChirpDatabaseConnection' not found.");
 
 //ChirpDBContext created with our database path - which is specified in appsettings.json
 builder.Services.AddDbContext<ChirpDBContext>(options => options.UseSqlite(connectionString));
 
-//builder.Services.AddDefaultIdentity<Author>(options => options.SignIn.RequireConfirmedAccount = true)
-//.AddEntityFrameworkStores<ChirpDBContext>();
+var dbcon = new SqliteConnection(connectionString);
+//await dbcon.OpenAsync();
```

Comment:
Do we want this and below?

Reply:
Hmmm, I will remove the //AddDefaultIdentity comment, but I am not sure about the builder.Services.AddDatabaseDeveloperPageExceptionFilter() line. I'll copy the lines to discord and then delete it here

Reply:
Forgot to answer about the //await dbcon.OpenAsync(); part ^^' 
At the moment it's been commented out, but I have a feeling we might need it later (although since it says _Open_ we probably need something to _close_ again). I would like to leave it in for now, but I can also remove it and leave it in discord for later, if thats better?